.circleci/config.yml

```diff
@@ -243,3 +286,8 @@ workflows:
   ios-integration-test:
     jobs:
       - ios-integration-test: *release-tags-and-branches
+
+  deploy:
+    jobs:
+      - make-release: *release-tags
+      - prepare-next-version: *release-tags
```

Comment:
Hmm this is setup as a single workflow in iOS and android. Should we update those as well? Also, I believe you can run failed jobs in a workflow: 
![image](https://user-images.githubusercontent.com/808417/183691679-457f4813-3ae2-43a6-abee-94998b648d78.png)
Does that have any issues that you know of?

Reply:
Leave it like this. I still don't understand what "from Failed" means, since I believe sometimes I see it runs even succeeded jobs but I just tested it and it works as expected. Don't worry about it.

Reply:
My guess is that it would start executing failed jobs, and any jobs that depend on those... That said, I'm not sure either 🤷‍♂️